### ISSUE TITLE
is there a way of setting up the bitnami/kong with GCP cloud sql postgres db

### ISSUE THREAD

**Issue by kevinchu85** [bitnami/charts#3360]
I had a look at the bitnami/kong deployment and it got me thinking the Kong DB which is on a postgres DB container. The issue with that is once the size of the Persistent volume is set it cannot be expanded.

I was wondering if there is a way to create a postgrees db on GCP cloud sql and deploy kong with connection to cloud sql?

**Comment by carrodher** [comment]
Kong is able to use PostgreSQL or Cassandra as databases, in this case, the bitnami/kong chart is able to work with both options. In the same way, each option can be deployed with a bundled PostgreSQL or Cassandra subchart that can be enabled with some parameters, and also, the bitnami/kong chart can be connected to an external database deployed in any way externally to the chart. [Here](https://github.com/bitnami/charts/blob/master/bitnami/kong/README.md#database-backend) you can find more information about this topic.

**Comment by kevinchu85** [comment]
@carrodher Thanks for pointing that out I think that is where I went wrong I was passing the external db host, dbuser and password but I was also using --set postgresql.enabled=true.

Thank you let me try again with false and see how it goes

**Comment by kevinchu85** [comment]
@carrodher I just created an sql instance and a database by the name of Kong. But it is an empty shell right now, do I need to configure the db schema for it as well? or the deployment will create the schema on the db?

**Comment by carrodher** [comment]
> But it is an empty shell right now, do I need to configure the db schema for it as well?

I don't think so, the PostgreSQL included as subchart doesn't contain any pre-configuration, is the regular Bitnami PostgreSQL chart without specific changes related to Kong.

> and I think it is not that straight forward to connect a container in GKE to a db that is not a part of the GKE Cluster?

You need to make the db accessible between both "networks" and use the user/password parameters to secure the connection, I am not totally sure how to manage this kind of things in the different Cloud Providers. Happy to help if you have any question related to the chart itself
